src/PriceCalculatorKata/Utils/ProductReport.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Globalization;
+
+namespace PriceCalculatorKata
+{
+    class ProductReport
+    {
+        public RegionInfo RegionInfo { private get; set; }
```

Comment:
Maybe you need to set a default value here, because anyone could call `ReportCostByName` without setting this value.